src/test/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarityITCase.java

```diff
@@ -53,8 +54,7 @@
  * See also http://maven.apache.org/surefire/maven-failsafe-plugin/examples/single-test.html
  */
 public class VariantSimilarityITCase {
-  
-  static final String OUTPUT_SUFFIX = "-00000-of-00001";
```

Comment:
The test started outputting 17 shards for the cloud case (although only one actually has content), so I adjusted the tearDown and read bits, and removed the setUp, since we no longer know beforehand exactly what files it will write.